src/Aspire.Hosting/build/Aspire.Hosting.targets

```diff
@@ -224,15 +224,24 @@ internal static class ]]>%(ClassName)<![CDATA[
 
   <PropertyGroup>
     <AspirePublisher Condition="'$(AspirePublisher)' == ''">manifest</AspirePublisher>
-    <AspireManifestPublishOutputPath Condition="'$(AspireManifestPublishOutputPath)' == ''">$(_AspireIntermediatePath)manifest.json</AspireManifestPublishOutputPath>
+    <AspireManifestPublishOutputPath Condition="'$(AspireManifestPublishOutputPath)' == ''">$(_AspireIntermediatePath)</AspireManifestPublishOutputPath>
   </PropertyGroup>
 
+  <Target Name="_EnsureAspireManifestPublishOutputPathExists" BeforeTargets="GenerateAspireManifest">
```

Comment:
You don't need both `DependsOnTargets` and `BeforeTargets`. You can just pick one. My preference would be `DependsOnTargets`.

Reply:
Oops, you are right, I started with BT but then switched to DOT and forgot to remove the older one. Fixed.